### PR TITLE
Issue #SB-13991 feat: added update available when content is updated

### PR DESCRIPTION
**Pull request by swetha-aggidevara**

### CODE REVIEW

File: server/controllers/content/content.ts [hunk 1]
```diff
@@ -21,6 +22,12 @@ export enum DOWNLOAD_STATUS {
     INDEXED = "DOWNLOADED",
     FAILED = "FAILED"
 }
+export enum CONTENT_UPDATE {
```
Comment (HarishGangula) [inline server/controllers/content/content.ts]: This is wrong,  check the name of the enum and what it contains
and everything should be consistent in enum https://www.typescriptlang.org/docs/handbook/enums.html
which follows standard all the values should be in uppercase
Why this is required ?

File: server/controllers/content/content.ts [hunk 2]
```diff
@@ -484,4 +496,37 @@ export default class Content {
...
+                        await this.databaseSdk.update('content', offlineContent.identifier, offlineContent);
+                        resolve(offlineContent);
+                    } else {
+                        offlineContent.desktopAppMetadata.lastUpdateCheckedOn = Date.now();
+                        offlineContent.desktopAppMetadata.updatedOn = Date.now();
+                        await this.databaseSdk.update('content', offlineContent.identifier, offlineContent);
+                        resolve(offlineContent);
+                    }
+                } catch(err) {
+                    resolve(offlineContent);
```
Comment (HarishGangula) [inline server/controllers/content/content.ts:527]: Why we are not logging error here?

File: server/routes.spec.ts [hunk 3]
```diff
@@ -32,7 +32,7 @@ describe('All', () => {
             .expect(200)
             .end((err, res) => {
                 if (res.statusCode >= 500) { logger.error(err); return done(); }
-                if (err && res.stausCode >= 400) { expect.fail(); return done(err); };
+                if (err && res.stausCode >= 400) { expect.fail(); return done(); };
```
Comment (HarishGangula) [inline server/routes.spec.ts]: spelling mistake here **statusCode**

**Review by HarishGangula — CHANGES_REQUESTED** [review]
@swetha-aggidevara resolve all the review comments and let me know

File: server/controllers/content/content.ts (hunk 2 above)
Comment (swetha-aggidevara) [reply server/controllers/content/content.ts:527]: If there is no Internet we need to send same data

Comment (HarishGangula) [reply server/controllers/content/content.ts:527]: Yes, but how I will know what is the error in logs if I want to debug?

**Review by HarishGangula — APPROVED** [review]

**Pull request merged**